pkgs/applications/radio/urh/default.nix

```
@@ -12,14 +13,15 @@ python3Packages.buildPythonApplication rec {
     sha256 = "1jrrj9c4ddm37m8j0g693xjimpnlvx7lan5kxish5p14xpwdak35";
   };
 
-  buildInputs = [ hackrf rtl-sdr airspy limesuite ];
+  buildInputs = [ hackrf rtl-sdr airspy limesuite libiio ]
+    ++ lib.optional USRPSupport uhd;
```

Nitpick: An additional empty line increases readability in that case:
```suggestion
    ++ lib.optional USRPSupport uhd;
    
```

---

done